ofl/danfo/DESCRIPTION.en_us.html

```diff
@@ -0,0 +1,3 @@
+<p>Danfo is an ELSH axis variable font with three main styles at 0,50, and 100. Danfo covers all of the Google SSA character set. The three styles reflect different cut-out shapes inspired by the vinyl cut-out lettering of the public transportation buses in Lagos, Nigeria.</p>
```

Comment:
Nit. I don't think most users would know what the abbreviations ELSH and SSA mean. Maybe nicer to em